watertap/unit_models/translators/translator_adm1_asm2d.py

```diff
@@ -96,8 +122,49 @@ def build(self):
         # Call UnitModel.build to setup dynamics
         super(TranslatorDataADM1ASM2D, self).build()
 
+        eps = 0
+        mw_p = 31 * pyunits.kg / pyunits.kmol
+        mw_n = 14 * pyunits.kg / pyunits.kmol
         mw_c = 12 * pyunits.kg / pyunits.kmol
 
+        self.f_sI_xc = Param(
```

Comment:
These factors look very familiar... are you sure these don't already live on ASM2D or ADM1 reaction or prop models?

Reply:
They live in the ADM1 reaction models, but I'm not sure of the correct syntax for implementing it here. I assumed it would be `blk.config.inlet_reaction_package.f_sI_xc`, but that's not correct. If we do update this, we should also update it in the ASM2d/ADM1 translator as well.